api-server/api.js

```diff
@@ -3,25 +3,24 @@
 
 module.exports = function api( options ) {
   var seneca = this
-
+  
```

Comment:
the file seems to have spaces. Please remove (and set editor to automatically remove)